src/bci_build/package/mariadb/README.server.md.j2

```diff
@@ -0,0 +1,68 @@
+* Create a data directory on the host system (outside the container) and mount this to a directory visible from inside the container. This places the database files in a known location on the host system, and makes it easy for tools and applications on the host system to access the files. The downside is that the user needs to make sure that the directory exists, and that e.g. directory permissions and other security mechanisms on the host system are set up correctly.
+
+To mount a host directory as a volume for you data simply run:
+
+```ShellSession
+$ podman run -it --rm -v /my/own/datadir:/var/lib/mysql:Z -p 3306:3306 -e MARIADB_ROOT_PASSWORD=my-password {{reference}}
+```
+
+The `-v /my/own/datadir:/var/lib/mysql:Z` part of the command mounts the `/my/own/datadir` directory from the underlying host system as `/var/lib/mysql` inside the container, where MariaDB by default will write its data files.
+
+## Environment Variables
+
+One of `MARIADB_RANDOM_ROOT_PASSWORD`, `MARIADB_ROOT_PASSWORD_HASH`, `MARIADB_ROOT_PASSWORD` or `MARIADB_ALLOW_EMPTY_ROOT_PASSWORD` (or equivalents, including `*_FILE`), is required.
+
+All other environment variables are optional.
+
+The complete list of environment variables is documented on MariaDB's Knowledge Base [MariaDB Server Docker Official Image Environment Variables](https://mariadb.com/kb/en/mariadb-server-docker-official-image-environment-variables/).
+
+## Health, Liveness and Readiness
+
+There's no explicit health checks added to the container image. However, you can use the `healthcheck.sh` script to choose from a (non-exhaustive) list of tests to check for whatever you consider health/liveness/readiness.
+
+If there is no database initialized when the container starts, then a default database will be created. This means that incoming connections will not be accepted until the initialization completes. Use the `healthcheck.sh` script to check for container readiness.
+
+To learn about `healthcheck.sh`, how to use it and which exact tests are provided check MariaDB's Knowledge Base [Using Healthcheck.sh](https://mariadb.com/kb/en/using-healthcheck-sh/).
+
+## Usage against an existing database
+
+If you start your MariaDB container instance with a data directory that already contains a database (specifically, a MySQL subdirectory), no environment variables that control initialization will be needed or examined, and no pre-existing databases will be changed.
+
+The only exception is the `MARIADB_AUTO_UPGRADE` environment variable, that if set, might cause `mysql_upgrade` or `mariadb-upgrade` to run, which might change the system tables.
```

Comment:
```suggestion
The only exception is the `MARIADB_AUTO_UPGRADE` environment variable. If it's set, it may run `mysql_upgrade` or `mariadb-upgrade`, potentially causing changes to the system tables.
```